src/core/plugins/oas3/components/request-body.jsx

```diff
@@ -83,7 +83,20 @@ const RequestBody = ({
 
   const mediaTypeValue = requestBodyContent.get(contentType, OrderedMap())
   const schemaForMediaType = mediaTypeValue.get("schema", OrderedMap())
-  const examplesForMediaType = mediaTypeValue.get("examples", null)
+  let examplesForMediaType = mediaTypeValue.get("examples", null)
+  if(examplesForMediaType) {
+    examplesForMediaType = examplesForMediaType.map((container, key) => {
```

Comment:
We should create a new const, to reflect that it's a mapped value from source. Based on the changes, I would rename the above/original `examplesForMediaType`. In the case that the renamed const doesn't exist, and assign the fallback value to the (new) `examplesForMediaType`

Reply:
Renamed old to `rawExamplesOfMediaType` and the actual example that is prepared for the media type is now named `sampleForMediaType`. I think this will clarify the meaning of the values.